hx_lti_initializer/views.py

```diff
@@ -213,6 +213,16 @@ def launch_lti(request):
         object_id = get_lti_value(settings.LTI_OBJECT_ID, tool_provider)
         course_id = str(course)
         if set(roles) & set(settings.ADMIN_ROLES):
+            try:
+                userfound = LTICourseAdmin.objects.get(
```

Comment:
Sounds good and yeah there was quite a bit of pushback from the folks here at Hx in having the learner's emails be in a server outside of the edX platform so I think we would prefer that unless we can make a good case as to why not.